src/CONST.js

```diff
@@ -72,6 +72,7 @@ const CONST = {
         },
         REGEX: {
             IBAN: /^[A-Za-z0-9]{2,30}$/,
```

Comment:
IBAN regex isn't used anywhere. We're safe to remove it.

Reply:
Yeah we can remove it.